rust/parquet/src/reader/schema.rs

```diff
@@ -808,4 +824,41 @@ mod tests {
             assert_eq!(arrow_fields[i], converted_fields[i]);
         }
     }
+
+    #[test]
```

Comment:
This looks good @liurenjie1024, but can you please also add temporal data types to the `message_type`? And what about structs and lists, or shall we cover that separately/later?

Reply:
ColumnDesc is used for leaf types in parquet. I think we need to add test for repeated type and temporal data, but no need for structs.

Reply:
Tests added.